CONTRIBUTNG.md

```diff
@@ -1,54 +0,0 @@
-# Contributing
-When bringing people on as contributors we prefer that they start with an issue tagged "good first issue." After a contributor has shown they're a good fit and have completed 1 or 2 issues labeled "good first issue" with pull requests that have been accepted feel free of moving on to other issues. We're always thrilled to see new contributors. Once an open source contributor has completed an issue labeled "good first issue" they are eligible to apply for and receive reward DAO funds.
-
-##
-
-## Setup
-In order to get the repository setup before contributions refer to the README.md file.
-
-## Contribution Steps
-
-1. *Find a suitable issue*
-
-    If this is your first time look for an issue labeled "good first issue" otherwise look for issues labeled help wanted. If another issue jumps out at you please engage on the issue before starting it.

```

Comment:
If this is your first time contributing look for issues labeled "good first issue", otherwise look for issues labeled "help wanted". If another issue jumps out at you please engage on the issue before starting it.